dev/app/main.jsx

```diff
@@ -1,6 +1,4 @@
-'use strict';
```

Comment:
Turns out `strict` mode is default for modules -- so we don't need to do this anymore.